templates/coolhole-coolpoints.pug

```diff
@@ -54,37 +54,33 @@ mixin coolpointsusertable
 
 mixin coolpointsuserprompt
     #coolPointsPrompt.modal.fade(role="dialog", aria-hidden="true", tabindex="-1")
-      .modal-dialog.modal-lg(role="document")
-        .modal-content
-          .modal-header.pb-0.mb-0
-            .row.modal-header-row
-              .header-icon-container.display-inline.mt-1.ml-3.pl-2
-                svg.ch-icon.header-icon.mt-n2: use(xlink:href="#ch-icon-ui-coin-stencil")
-              .col
-                h4.modal-title.mt-n1.mb-2 CoolPoints
+      .modal-dialog.modal-lg#cpModal(role="document")
+        .modal-content#cpModalContent
+          .modal-header#cpModalHeader
+            svg#cpModalIcon: use(xlink:href="#ch-icon-ui-coin-stencil")
+            h2.modal-title: i(style="font-size: 48px; font-weight: 800") CoolPoints
             button.close(data-dismiss="modal", aria-label="Close"): span(aria-hidden="true") &times;
-          .modal-body.mt-0.pt-0.px-0.pb-2
+          .modal-body#cpModalBody
             ul#cpTab.nav.nav-tabs.nav-justified(role="tablist")
-              li.nav-item: a#cpPromptTab.nav-link.active(data-toggle="tab", href="#cp-prompt", role="tab", aria-controls="cp-prompt", aria-selected="true") Our mission
+              li.nav-item: a#cpPromptTab.tab-pane.active(data-toggle="tab", href="#cp-prompt", role="tab", aria-controls="cp-prompt", aria-selected="true") Our mission
               li.nav-item: a#cpStatusOfActionsTab.nav-link(data-toggle="tab", href="#cp-status-of-actions", role="tab", aria-controls="cp-status-of-actions", aria-selected="false") Status of Actions
               li.nav-item: a#cpCasinoTab.nav-link(data-toggle="tab", href="#cp-casino", role="tab", aria-controls="cp-casino", aria-selected="false") Casino
-            .container-fluid.mt-2.px-3
+            .container
               #cpTabContent.tab-content.row.pt-2
                 +coolpoints-prompt
                 +status-of-actions
                 +casino
 
 mixin coolpoints-prompt
   #cp-prompt.tab-pane.fade.col-12.active.in(role="tabpanel", aria-labelledby="cpPromptTab")
```

Comment:
nit: I'd suggest making a common class for each tab content for this prompt. I think some of these general styles like `    .col(style="justify-content: space-evenly; align-items: center;")` could probably be wrapped up in that

Reply:
Github hid this one for some reason...
Anyways, I agree and now I'm thinking of just restructuring the modal to match the other modals for easier CSS changes.